docs/src/pages/milestones.tsx

```diff
@@ -0,0 +1,407 @@
+import {
+  mdiAccountGroup,
+  mdiAndroid,
+  mdiAppleIos,
+  mdiArchiveOutline,
+  mdiBookSearchOutline,
+  mdiCheckAll,
+  mdiCheckboxMarked,
+  mdiCollage,
+  mdiDevices,
+  mdiFaceMan,
+  mdiFaceManOutline,
+  mdiFile,
+  mdiFileSearch,
+  mdiFolder,
+  mdiHeart,
+  mdiImage,
+  mdiImageAlbum,
+  mdiImageMultipleOutline,
+  mdiImageSearch,
+  mdiKeyboardSettingsOutline,
+  mdiMagnify,
+  mdiMap,
+  mdiMaterialDesign,
+  mdiMerge,
+  mdiMonitor,
+  mdiMotionPlayOutline,
+  mdiPanVertical,
+  mdiPartyPopper,
+  mdiRaw,
+  mdiRotate360,
+  mdiSecurity,
+  mdiServer,
+  mdiShareAll,
+  mdiShareCircle,
+  mdiStar,
+  mdiTag,
+  mdiText,
+  mdiThemeLightDark,
+  mdiTrashCanOutline,
+  mdiVideo,
+  mdiWeb,
+} from '@mdi/js';
+import Layout from '@theme/Layout';
+import React from 'react';
+import Timeline from '../components/timeline';
+
+interface Item {
+  icon: string;
+  title: string;
+  description?: string;
+  release: string;
+}
+
+const items: Item[] = [
```

Comment:
I know this makes it a lot more complex, but what do you think of a "Milestone" tag on PRs? Then, this list could be dynamic. (Maybe fetch milestones once per hour or something) However, getting the release would be a bit of a pain this way.

Reply:
Maybe. There are downsides with that approach as well. Potentially not a static build anymore. I'm not sure what it would take to pre-render it, not a react expert here. I also like the icons and PRs don't have that right now. I think the little bit of manual effort to maintain this is worth the simplicity.